.changeset/slimy-berries-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@atproto/lex-cli": patch
+---
+
+Export record types using the `Main` identifier
```

Comment:
We probably also want a patch changeset for the @atproto/api package, as I believe codegen ends up being part of its public interface.